crates/matrix-sdk-common/Cargo.toml

```diff
@@ -50,10 +50,11 @@ wasm-bindgen-futures = { version = "0.4.33", optional = true }
 web-sys = { workspace = true, features = ["console"] }
 
 [dev-dependencies]
-assert_matches.workspace = true
+assert_matches = { workspace = true }
+assert_matches2. workspace = true
+proptest.workspace = true
```

Comment:
Why the move of the proptest import here? This should ideally be sorted alphabetically.